api/app/controllers/spree/api/v2/storefront/stores_controller.rb

```diff
@@ -7,8 +7,21 @@ def current
             render_serialized_payload { serialize_resource(current_store) }
           end
 
+          def export_google_rss_feed
+            send_data export_google_rss_service.value[:file], filename: 'products.rss', type: 'text/xml'
+          end
+
           private
 
+          def update_settings
```

Comment:
```suggestion
          def settings
```
because this method don't update anything and name can be confusing